### PR TITLE
Added hover effects on issues to grab section cards

### PR DESCRIPTION
**Pull request by cleverhare**
As I have generate an Issue

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,8 +29,7 @@
     <!-- Theme CSS -->
     <link href="css/agency.css" rel="stylesheet">
     <link rel="stylesheet" href="css/community.css">
-
-
+    <link rel="stylesheet" href="./css/cleverhare.css">
```
Comment (AmanSarraf) [inline index.html:32]: Please rename the CSS file to something more relevant, that might not result in any future fellow contributors
For Example : 
````suggestion
    <link rel="stylesheet" href="./css/Issues_hover.css">
````

**Review by AmanSarraf — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (cleverhare) [reply index.html:32]: Sure

**Comment by cleverhare** [comment]
Done

File: index.html [hunk 2]
```diff
@@ -29,8 +29,7 @@
     <!-- Theme CSS -->
     <link href="css/agency.css" rel="stylesheet">
     <link rel="stylesheet" href="css/community.css">
-
-
+    <link rel="stylesheet" href="./css/Issues_hover.css">
```
Comment (AmanSarraf) [inline index.html:32]: need to rename the css file name as well , check It's cleverhare.css  should be >>>>>>> Issues_hover.css instead.
else the stylesheet wont be linked.
Also update your fork before raising PR,The latest branch is updated last night.

**Review by AmanSarraf — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (cleverhare) [reply index.html:32]: Okay brother

**Comment by cleverhare** [comment]
Generated another one

**Pull request closed without merging**